lang/en/block_iagora.php

```diff
@@ -22,4 +22,8 @@
  * @license   http://www.gnu.org/copyleft/gpl.html GNU GPL v3 or later
  */
 
-$string['pluginname'] = 'IAGORA';
+ $string['iframe_url'] = 'Iframe URL';
+ $string['iframe_url_desc'] = 'Public URL available for chat iframe';
```

Comment:
```suggestion
 $string['iframeurl'] = 'Iframe URL';
 $string['iframeurl_help'] = 'Public URL available for the Microsoft Copilot chat iframe';
```
It seems the "addHelpButton" Moodle API works with a "name_help" convention. See: https://docs.moodle.org/dev/lib/formslib.php_Form_Definition#addHelpButton

Reply:
added